### ISSUE TITLE
Conflict with Tom's Simple Storage Mod

### ISSUE THREAD

**Issue by OakPlatinum** [Sollace/Unicopia#441]
When a jar is placed beside an inventory connector in Tom's, the game crashes. It Will crash again if you try to load the chunk. No crash report because I think it's easy to be repeated. Unicopia: 1.3.8+1.20.4. Tom's: 1.20.4-1.6.14.

**Comment by Sollace** [comment]
Please add the crash report. It's not about repeating it, it's about seeing what the cause was. I'm not going to go through the trouble of installing Tom's into my dev environment to maybe reproduce the issue

**Comment by OakPlatinum** [comment]
Wow, so fast.

Okay, but it will be tomorrow. And I'm not sure if I have deleted it.

**Comment by OakPlatinum** [comment]
I didn't stupidly deleted it. :)
```
---- Minecraft Crash Report ----
// I let you down. Sorry :(

Time: 2024-08-17 19:29:42
Description: Ticking block entity

java.lang.IndexOutOfBoundsException: Index 15 out of bounds for length 15
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:361)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at net.fabricmc.fabric.impl.transfer.item.SidedInventoryStorageImpl.createWrapperList(SidedInventoryStorageImpl.java:53)
	at net.fabricmc.fabric.impl.transfer.item.SidedInventoryStorageImpl.<init>(SidedInventoryStorageImpl.java:38)
	at net.fabricmc.fabric.impl.transfer.item.InventoryStorageImpl.getSidedWrapper(InventoryStorageImpl.java:110)
	at net.fabricmc.fabric.impl.transfer.item.InventoryStorageImpl.of(InventoryStorageImpl.java:65)
	at net.fabricmc.fabric.api.transfer.v1.item.InventoryStorage.of(InventoryStorage.java:62)
	at net.fabricmc.fabric.api.transfer.v1.item.ItemStorage.lambda$static$2(ItemStorage.java:109)
	at net.fabricmc.fabric.impl.lookup.block.BlockApiLookupImpl.find(BlockApiLookupImpl.java:99)
	at com.tom.storagemod.tile.InventoryConnectorBlockEntity.updateServer(InventoryConnectorBlockEntity.java:90)
	at com.tom.storagemod.util.TickerUtil.lambda$createTicker$1(TickerUtil.java:10)
	at net.minecraft.class_2818$class_5563.method_31703(class_2818.java:662)
	at net.minecraft.class_2818$class_5564.method_31703(class_2818.java:716)
	at net.minecraft.class_1937.method_18471(class_1937.java:482)
	at net.minecraft.class_3218.method_18765(class_3218.java:404)
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:948)
	at net.minecraft.class_3176.method_3813(class_3176.java:283)
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:845)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:683)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:270)
	at java.base/java.lang.Thread.run(Thread.java:833)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:361)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at net.fabricmc.fabric.impl.transfer.item.SidedInventoryStorageImpl.createWrapperList(SidedInventoryStorageImpl.java:53)
	at net.fabricmc.fabric.impl.transfer.item.SidedInventoryStorageImpl.<init>(SidedInventoryStorageImpl.java:38)
	at net.fabricmc.fabric.impl.transfer.item.InventoryStorageImpl.getSidedWrapper(InventoryStorageImpl.java:110)
	at net.fabricmc.fabric.impl.transfer.item.InventoryStorageImpl.of(InventoryStorageImpl.java:65)
	at net.fabricmc.fabric.api.transfer.v1.item.InventoryStorage.of(InventoryStorage.java:62)
	at net.fabricmc.fabric.api.transfer.v1.item.ItemStorage.lambda$static$2(ItemStorage.java:109)
	at net.fabricmc.fabric.impl.lookup.block.BlockApiLookupImpl.find(BlockApiLookupImpl.java:99)
	at com.tom.storagemod.tile.InventoryConnectorBlockEntity.updateServer(InventoryConnectorBlockEntity.java:90)
	at com.tom.storagemod.util.TickerUtil.lambda$createTicker$1(TickerUtil.java:10)
	at net.minecraft.class_2818$class_5563.method_31703(class_2818.java:662)
	at net.minecraft.class_2818$class_5564.method_31703(class_2818.java:716)
	at net.minecraft.class_1937.method_18471(class_1937.java:482)

-- Block entity being ticked --
Details:
	Name: toms_storage:ts.inventory_connector.tile // com.tom.storagemod.tile.InventoryConnectorBlockEntity
	Block: Block{toms_storage:ts.inventory_connector}
	Block location: World: (-275,69,-1507), Section: (at 13,5,13 in -18,4,-95; chunk contains blocks -288,-64,-1520 to -273,319,-1505), Region: (-1,-3; contains chunks -32,-96 to -1,-65, blocks -512,-64,-1536 to -1,319,-1025)
	Block: Block{toms_storage:ts.inventory_connector}
	Block location: World: (-275,69,-1507), Section: (at 13,5,13 in -18,4,-95; chunk contains blocks -288,-64,-1520 to -273,319,-1505), Region: (-1,-3; contains chunks -32,-96 to -1,-65, blocks -512,-64,-1536 to -1,319,-1025)
Stacktrace:
	at net.minecraft.class_2818$class_5563.method_31703(class_2818.java:662)
	at net.minecraft.class_2818$class_5564.method_31703(class_2818.java:716)
	at net.minecraft.class_1937.method_18471(class_1937.java:482)
	at net.minecraft.class_3218.method_18765(class_3218.java:404)
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:948)
	at net.minecraft.class_3176.method_3813(class_3176.java:283)
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:845)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:683)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:270)
	at java.base/java.lang.Thread.run(Thread.java:833)

-- Affected level --
Details:
	All players: 1 total; [class_3222['OakPlatinum'/183, l='ServerLevel[world]', x=-274.57, y=70.00, z=-1505.67]]
	Chunk stats: 6296
	Level dimension: minecraft:overworld
	Level spawn location: World: (0,119,0), Section: (at 0,7,0 in 0,7,0; chunk contains blocks 0,-64,0 to 15,319,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Level time: 64640740 game time, 1107695 day time
	Level name: world
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Level weather: Rain time: 3665 (now: false), thunder time: 53476 (now: false)
	Known server brands: fabric, fabric arclight/Whisper
	Removed feature flags: 
	Level was modded: true
	Level storage version: 0x04ABD - Anvil
	Loaded entity count: 458
Stacktrace:
	at net.minecraft.class_3218.method_8538(class_3218.java:1681)
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:951)
	at net.minecraft.class_3176.method_3813(class_3176.java:283)
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:845)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:683)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:270)
	at java.base/java.lang.Thread.run(Thread.java:833)

-- System Details --
Details:
	Minecraft Version: 1.20.4
	Minecraft Version ID: 1.20.4
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.7, Eclipse Adoptium
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Eclipse Adoptium
	Memory: 10879411224 bytes (10375 MiB) / 12582912000 bytes (12000 MiB) up to 12582912000 bytes (12000 MiB)
	CPUs: 32
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 9 5950X 16-Core Processor            
	Identifier: AuthenticAMD Family 25 Model 33 Stepping 0
	Microarchitecture: Zen 3
	Frequency (GHz): 3.40
	Number of physical packages: 1
	Number of physical CPUs: 16
	Number of logical CPUs: 32
	Graphics card #0 name: OrayIddDriver Device
	Graphics card #0 vendor: Shanghai Best Oray Information Technology Co., Ltd.
	Graphics card #0 VRAM (MB): 0.00
	Graphics card #0 deviceId: unknown
	Graphics card #0 versionInfo: DriverVersion=17.1.58.818
	Graphics card #1 name: NVIDIA GeForce RTX 3080 Ti
	Graphics card #1 vendor: NVIDIA (0x10de)
	Graphics card #1 VRAM (MB): 4095.00
	Graphics card #1 deviceId: 0x2208
	Graphics card #1 versionInfo: DriverVersion=32.0.15.6070
	Memory slot #0 capacity (MB): 32768.00
	Memory slot #0 clockSpeed (GHz): 3.20
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 32768.00
	Memory slot #1 clockSpeed (GHz): 3.20
	Memory slot #1 type: DDR4
	Memory slot #2 capacity (MB): 32768.00
	Memory slot #2 clockSpeed (GHz): 3.20
	Memory slot #2 type: DDR4
	Memory slot #3 capacity (MB): 32768.00
	Memory slot #3 clockSpeed (GHz): 3.20
	Memory slot #3 type: DDR4
	Virtual memory max (MB): 130977.13
	Virtual memory used (MB): 64222.66
	Swap memory total (MB): 0.00
	Swap memory used (MB): 0.00
	JVM Flags: 2 total; -Xms12000M -Xmx12000M
	Fabric Mods: 
		allmusic_server: AllMusic Server 3.1.2
			com_squareup_okhttp3_okhttp: okhttp 4.10.0
			com_squareup_okio_okio-jvm: okio-jvm 3.0.0
			org_xerial_sqlite-jdbc: sqlite-jdbc 3.39.3.0
		appleskin: AppleSkin 2.5.1+mc1.20.3
		architectury: Architectury 11.1.17
		artifacts: Artifacts 10.0.11
			cardinal-components-base: Cardinal Components API (base) 5.4.0
			cardinal-components-entity: Cardinal Components API (entities) 5.4.0
			expandability: ExpandAbility 10.0.1
			step-height-entity-attribute: Step Height Entity Attribute 1.2.0
		balm-fabric: Balm 9.0.8
		beautify: Beautify 1.2.0+1.20.4
		betterdeserttemples: YUNG's Better Desert Temples 1.20.4-Fabric-3.4.4
		betterdungeons: YUNG's Better Dungeons 1.20.4-Fabric-4.4.4
		betterendisland: YUNG's Better End Island 1.20.4-Fabric-2.4.3
		betterfortresses: YUNG's Better Nether Fortresses 1.20.4-Fabric-2.4.4
		betterjungletemples: YUNG's Better Jungle Temples 1.20.4-Fabric-2.4.4
		bettermineshafts: YUNG's Better Mineshafts 1.20.4-Fabric-4.4.2
		betteroceanmonuments: YUNG's Better Ocean Monuments 1.20.4-Fabric-3.4.3
		betterstrongholds: YUNG's Better Strongholds 1.20.4-Fabric-4.4.3
		betterwitchhuts: YUNG's Better Witch Huts 1.20.4-Fabric-3.4.3
		bigpony: Big Pony 1.11.2+1.20.4
		c2me: Concurrent Chunk Management Engine 0.2.0+alpha.11.67
			c2me-base: Concurrent Chunk Management Engine (Base) 0.2.0+alpha.11.67
			c2me-fixes-chunkio-threading-issues: Concurrent Chunk Management Engine (Fixes/Chunk IO/Threading Issues) 0.2.0+alpha.11.67
			c2me-fixes-general-threading-issues: Concurrent Chunk Management Engine (Fixes/General/Threading Issues) 0.2.0+alpha.11.67
			c2me-fixes-worldgen-threading-issues: Concurrent Chunk Management Engine (Fixes/WorldGen/Threading Issues) 0.2.0+alpha.11.67
			c2me-fixes-worldgen-vanilla-bugs: Concurrent Chunk Management Engine (Fixes/WorldGen/Vanilla Bugs) 0.2.0+alpha.11.67
			c2me-notickvd: Concurrent Chunk Management Engine (No Tick View Distance) 0.2.0+alpha.11.67
			c2me-opts-allocs: Concurrent Chunk Management Engine (Optimizations/Memory Allocations) 0.2.0+alpha.11.67
			c2me-opts-chunk-access: Concurrent Chunk Management Engine (Optimizations/Chunk Access) 0.2.0+alpha.11.67
			c2me-opts-chunkio: Concurrent Chunk Management Engine (Optimizations/Chunk IO) 0.2.0+alpha.11.67
			c2me-opts-math: Concurrent Chunk Management Engine (Optimizations/Math) 0.2.0+alpha.11.67
			c2me-opts-scheduling: Concurrent Chunk Management Engine (Optimizations/Scheduling) 0.2.0+alpha.11.67
			c2me-opts-worldgen-general: Concurrent Chunk Management Engine (Optimizations/General WorldGen) 0.2.0+alpha.11.67
			c2me-opts-worldgen-vanilla: Concurrent Chunk Management Engine (Optimizations/Vanilla WorldGen) 0.2.0+alpha.11.67
			c2me-rewrites-chunk-serializer: Concurrent Chunk Management Engine (Rewrites/Chunk Serializer) 0.2.0+alpha.11.67
			c2me-rewrites-chunkio: Concurrent Chunk Management Engine (Rewrites/Chunk IO) 0.2.0+alpha.11.67
			c2me-server-utils: Concurrent Chunk Management Engine (Server Utils) 0.2.0+alpha.11.67
			c2me-threading-chunkio: Concurrent Chunk Management Engine (Threading/WorldGen) 0.2.0+alpha.11.67
			c2me-threading-lighting: Concurrent Chunk Management Engine (Threading/Lighting) 0.2.0+alpha.11.67
			c2me-threading-worldgen: Concurrent Chunk Management Engine (Threading/WorldGen) 0.2.0+alpha.11.67
			com_ibm_async_asyncutil: asyncutil 0.1.0
			net_objecthunter_exp4j: exp4j 0.4.8
			org_threadly_threadly: threadly 7.0
		cardinal-components: Cardinal Components API 5.4.0
			cardinal-components-block: Cardinal Components API (blocks) 5.4.0
			cardinal-components-chunk: Cardinal Components API (chunks) 5.4.0
			cardinal-components-item: Cardinal Components API (items) 5.4.0
			cardinal-components-level: Cardinal Components API (world saves) 5.4.0
			cardinal-components-scoreboard: Cardinal Components API (scoreboard) 5.4.0
			cardinal-components-world: Cardinal Components API (worlds) 5.4.0
		chatimage: ChatImage 1.2.4+1.20.4+fabric
			actionlib: ActionLib 1.0.2-fabric
			io_github_kituin_chatimagecode: ChatImageCode 0.9.7
		chunky: Chunky 1.3.146
		clockin: Clock In 0.10.0
			completeconfig-base: completeconfig-base 2.4.0
			mysql_mysql-connector-java: mysql-connector-java 5.1.49
		cloth-config: Cloth Config v13 13.0.121
			cloth-basic-math: cloth-basic-math 0.6.1
		clumps: Clumps 15.0.0.2
		diamondchestshop: Diamond Chest Shop 1.2.6+1.20
		diamondeconomy: Diamond Economy 1.6.0+1.20.2
		emi: EMI 1.1.6+1.20.4+fabric
		enchantinginfuser: Enchanting Infuser 20.4.1
		endrem: End Remastered 5.2.4
		extended_drawers: Extended Drawers 2.1.1+mc.1.20.4
			config_toolkit: ConfigToolkit 1.0.0
				io_determann_shadow-api-17: shadow-api-17 0.1.1
				net_fabricmc_javapoet: javapoet 0.1.0
			graphlib: Graph Lib 1.4.0+1.20.4
				libnetworkstack: Lib Network Stack 0.11.0-pre.1
			yet_another_config_lib_v3: YetAnotherConfigLib 3.3.1+1.20.4
				com_twelvemonkeys_common_common-image: common-image 3.10.0
				com_twelvemonkeys_common_common-io: common-io 3.10.0
				com_twelvemonkeys_common_common-lang: common-lang 3.10.0
				com_twelvemonkeys_imageio_imageio-core: imageio-core 3.10.0
				com_twelvemonkeys_imageio_imageio-metadata: imageio-metadata 3.10.0
				com_twelvemonkeys_imageio_imageio-webp: imageio-webp 3.10.0
				org_quiltmc_parsers_gson: gson 0.2.1
				org_quiltmc_parsers_json: json 0.2.1
		fabric-api: Fabric API 0.97.1+1.20.4
			fabric-api-base: Fabric API Base 0.4.37+78d798af4f
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.50+82b1bb3e4f
			fabric-biome-api-v1: Fabric Biome API (v1) 13.0.17+78d798af4f
			fabric-block-api-v1: Fabric Block API (v1) 1.0.17+3e2216cb4f
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.5+78d798af4f
			fabric-command-api-v1: Fabric Command API (v1) 1.2.42+f71b366f4f
			fabric-command-api-v2: Fabric Command API (v2) 2.2.21+78d798af4f
			fabric-commands-v0: Fabric Commands (v0) 0.2.59+df3654b34f
			fabric-containers-v0: Fabric Containers (v0) 0.1.87+df3654b34f
			fabric-content-registries-v0: Fabric Content Registries (v0) 5.0.16+78d798af4f
			fabric-convention-tags-v1: Fabric Convention Tags 1.5.11+78d798af4f
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.24+78d798af4f
			fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.1.5+b90db5744f
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 13.2.5+5c0133444f
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.63+6605d1a84f
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.6.2+09fc25014f
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.7.2+389931eb4f
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.75+df3654b34f
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.47+78d798af4f
			fabric-item-api-v1: Fabric Item API (v1) 2.3.0+bcdd12964f
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.0.27+ee30b13a4f
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.3.1+a67ffb5d4f
			fabric-loot-api-v2: Fabric Loot API (v2) 2.1.10+78d798af4f
			fabric-message-api-v1: Fabric Message API (v1) 6.0.6+78d798af4f
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.66+78d798af4f
			fabric-networking-api-v1: Fabric Networking API (v1) 3.1.8+2e5ac5484f
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 13.1.0+dba1195c4f
			fabric-particles-v1: Fabric Particles (v1) 1.1.8+78d798af4f
			fabric-recipe-api-v1: Fabric Recipe API (v1) 2.0.21+78d798af4f
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 4.0.20+ee30b13a4f
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.43+73761d2e4f
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.1.2+e761c6694f
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.3.15+78d798af4f
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.11.21+ee30b13a4f
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.56+78d798af4f
			fabric-transfer-api-v1: Fabric Transfer API (v1) 4.0.12+90f2d7b14f
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 5.0.15+78d798af4f
		fabric-language-kotlin: Fabric Language Kotlin 1.11.0+kotlin.2.0.0
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 2.0.0
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 2.0.0
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 2.0.0
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 2.0.0
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.24.0
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.8.1
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.8.1
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.6.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.6.3
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.6.3
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.6.3
		fabricloader: Fabric Loader 0.15.11
			mixinextras: MixinExtras 0.3.5
		fancymenu: FancyMenu 3.2.3
			com_github_keksuccino_japng: japng 0.5.3
			com_github_rtyley_animated-gif-lib-for-java: animated-gif-lib-for-java animated-gif-lib-1.7
		fmsia: FancyMenu | System Interactions Addon 1.0.1
		forgeconfigapiport: Forge Config API Port 20.4.3
			com_electronwill_night-config_core: core 3.6.6
			com_electronwill_night-config_toml: toml 3.6.6
		framework: Framework 0.7.6
			org_javassist_javassist: javassist 3.29.2-GA
		ftbchunks: FTB Chunks 2004.1.2
		ftbessentials: FTB Essentials 2001.2.2
		ftblibrary: FTB Library 2004.2.2
		ftbranks: FTB Ranks 2004.2.0
		ftbteams: FTB Teams 2004.1.2
		icommon-bundle: iCommonLib-bundle bundle
			icommon: iCommonLib 1.20.2 1.20.2
		jade: Jade 13.3.1
		java: OpenJDK 64-Bit Server VM 17
		jei: Just Enough Items 17.3.0.52
		kirin: Kirin UI 1.17.1+1.20.4
		konkrete: Konkrete 1.9.2
		ledger: Ledger 1.3.1
			fabric-permissions-api-v0: fabric-permissions-api 0.3.1
			server_translations_api: Server Translations API 2.2.0+1.20.3-rc1
				packet_tweaker: Packet Tweaker 0.5.0+1.20.2-rc1
		lithium: Lithium 0.12.1
		luckperms: LuckPerms 5.4.113
		melius-vanish: Vanish 1.5.4+1.20.4
		melody: Melody 1.0.3
		midnightlib: MidnightLib 1.5.3
		minecraft: Minecraft 1.20.4
		minimotd-fabric: MiniMOTD 2.1.0
			adventure-platform-fabric: adventure-platform-fabric 5.11.0-SNAPSHOT
				net_kyori_adventure-api: adventure-api 4.15.0
				net_kyori_adventure-key: adventure-key 4.15.0
				net_kyori_adventure-platform-api: adventure-platform-api 4.3.2
				net_kyori_adventure-text-logger-slf4j: adventure-text-logger-slf4j 4.15.0
				net_kyori_adventure-text-minimessage: adventure-text-minimessage 4.15.0
				net_kyori_adventure-text-serializer-ansi: adventure-text-serializer-ansi 4.15.0
				net_kyori_adventure-text-serializer-gson: adventure-text-serializer-gson 4.15.0
				net_kyori_adventure-text-serializer-json: adventure-text-serializer-json 4.15.0
				net_kyori_adventure-text-serializer-plain: adventure-text-serializer-plain 4.15.0
				net_kyori_ansi: ansi 1.0.3
				net_kyori_examination-api: examination-api 1.3.0
				net_kyori_examination-string: examination-string 1.3.0
				net_kyori_option: option 1.0.0
		nochatreports: No Chat Reports 1.20.4-v2.6.0
		patchouli: Patchouli 1.20.4-85-FABRIC
			fiber: fiber 0.23.0-2
		pehkui: Pehkui 3.8.3+1.14.4-1.21
			kanos_config: Kanos Config 0.4.1+1.14.4-1.19.4
		puzzleslib: Puzzles Lib 20.4.52
		quickbackupmulti: QuickBackupMulti 2.0.2
			com_h2database_h2: h2 2.2.224
			de_bwaldvogel_mongo-java-server: mongo-java-server 1.45.0
			de_bwaldvogel_mongo-java-server-core: mongo-java-server-core 1.45.0
			de_bwaldvogel_mongo-java-server-h2-backend: mongo-java-server-h2-backend 1.45.0
			dev_morphia_morphia_morphia-core: morphia-core 2.3.9
			org_mongodb_bson: bson 4.11.1
			org_mongodb_mongodb-driver-core: mongodb-driver-core 4.11.1
			org_mongodb_mongodb-driver-sync: mongodb-driver-sync 4.11.1
			org_quartz-scheduler_quartz: quartz 2.3.2
			org_yaml_snakeyaml: snakeyaml 2.0
		repurposed_structures: Repurposed Structures 7.3.5+1.20.4-fabric
		starlight: Starlight 1.1.3+fabric.f5dcd1a
		styledplayerlist: Styled Player List 3.3.0+1.20.3
			placeholder-api: Placeholder API 2.3.0+1.20.3
			player-data-api: Player Data API 0.4.0+1.20.3
			predicate-api: Predicate API 0.3.0+1.20.2
		toms_storage: Tom's Simple Storage Mod 1.6.14
		toms_trading_network: Tom's Trading Network 0.3.0
		tpaplusplus: TPA++ 1.6.0-1.20.x-RELEASE-CANDIDATE-1
		travelersbackpack: Traveler's Backpack fabric-1.20.4-9.4.6
			reach-entity-attributes: Reach Entity Attributes 2.5.0
		trinkets: Trinkets 3.8.1
		unicopia: Unicopia 1.3.8+1.20.4
			fabwork: Fabwork 1.3.0+1.20.2
			romanizer: Romanizer 1.0.2
			terraform-wood-api-v1: Terraform Wood API (v1) 8.0.0-beta.1
		veinmining: Vein Mining 3.1.1+1.20.4
			spectrelib: SpectreLib 0.15.2+1.20.4
		voicechat: Simple Voice Chat 1.20.4-2.5.15
		waystones: Waystones 16.0.5
		worldedit: WorldEdit 7.3.0+6678-55745ad
		xaerominimap: Xaero's Minimap 24.1.1
		xaeroworldmap: Xaero's World Map 1.38.4
		yungsapi: YUNG's API 1.20.4-Fabric-4.4.3
			org_reflections_reflections: reflections 0.10.2
		yungsbridges: YUNG's Bridges 1.20.4-Fabric-4.4.2
		yungsextras: YUNG's Extras 1.20.4-Fabric-4.4.2
	Server Running: true
	Player Count: 1 / 30; [class_3222['OakPlatinum'/183, l='ServerLevel[world]', x=-274.57, y=70.00, z=-1505.67]]
	Data Packs: vanilla, fabric, artifacts, balm-fabric (incompatible), beautify, betterdeserttemples, betterdungeons, betterendisland, betterfortresses, betterjungletemples, bettermineshafts, betteroceanmonuments, betterstrongholds, betterwitchhuts, clockin, enchantinginfuser (incompatible), endrem, extended_drawers, fabric-convention-tags-v1, ftbchunks, ftbessentials (incompatible), ledger, melius-vanish, patchouli (incompatible), repurposed_structures, server_translations_api, toms_storage (incompatible), toms_trading_network, travelersbackpack, trinkets, unicopia, veinmining (incompatible), waystones (incompatible), yungsbridges, yungsextras, file/Repurposed_Structures-Better_Desert_Temples_v8.zip (incompatible), file/Repurposed_Structures-Better_Dungeons_v5.zip (incompatible), file/Repurposed_Structures-Better_Monuments_v6.zip (incompatible), file/Repurposed_Structures-Better_Nether_Fortress_v6.zip (incompatible), file/Repurposed_Structures-Better_Strongholds_v5.zip (incompatible), file/Repurposed_Structures-Better_Witch_Huts_v4.zip (incompatible), file/vanillarefresh-v1.4.19h_1.20.x.zip
	Enabled Feature Flags: minecraft:vanilla
	World Generation: Experimental
	Is Modded: Definitely; Server brand changed to 'fabric'
	Type: Dedicated Server (map_server.txt)
```

**Comment by Sollace** [comment]
This looks to be more of a fabric api bug than an issue with the mods. The
jars implement support for the transfer api which Tom is going through to
read the contents and is what seems to be causing the problem.
